package.json

```
@@ -10,7 +10,9 @@
   "scripts": {
     "db:seed": "cd services/app-api && yarn run db:seed",
     "test": "cd tests/cypress && npm test && cd -",
-    "test:ci": "cd tests/cypress && yarn run test:ci"
+    "test:ci": "cd tests/cypress && yarn run test:ci",
+    "test:e2e": "yarn playwright test",
+    "test:e2e-ui": "yarn playwright test --ui"
```

Remove redundant lead `yarn` command
```suggestion
    "test:e2e": "playwright test",
    "test:e2e-ui": "playwright test --ui"
```